src/assets/scripts/client/aircraft/AircraftStripView.js

```diff
@@ -371,17 +371,14 @@ export default class AircraftStripView {
      * @param destinationText {string}
      * @param isFollowingSID {boolean}
      * @param isFollowingSTAR {boolean}
-     * @param fixRestrictions {object}
```

Comment:
This whole file should be deleted. Likely still around cause you had changes here I'm guessing.